helios.js

```diff
@@ -2985,7 +2985,7 @@ export class NetworkParams {
 	}
 	
 	get costModel() {
-		return assertDefined(this.#raw?.latestParams?.costModels?.PlutusScriptV2, "'obj.latestParams.costModels.PlutusScriptV2' undefined");
+		return assertDefined(this.#raw?.costModels?.PlutusScriptV2, "'obj.costModels.PlutusScriptV2' undefined");
```

Comment:
I shoud've probably documented this better: #raw is downloaded from https://d1t0d7c2nekuk0.cloudfront.net/preview.json (or preprod, or mainnet)

It isn't the params as returned by cardano-cli, nor as returned by Blockfrost

Reply:
The reason is that this object for convenience also includes the timeStamp and slotNumber of the last sync, so that the time -> slotNumber can be accurately calculated

Reply:
Got it. I can confirm these files are working with the previous version. I'll revert these changes.